src/components/application_manager/include/application_manager/rpc_handler_impl.h

```diff
@@ -172,6 +172,8 @@ class RPCHandlerImpl : public RPCHandler,
 
   hmi_apis::HMI_API hmi_so_factory_;
   mobile_apis::MOBILE_API mobile_so_factory_;
+  ns_smart_device_link_rpc::V1::v4_protocol_v1_2_no_extra
+      v4_protocol_so_factory_;
```

Comment:
@IGapchuk BTW, do we really need the copies of these huge objects in this rpc_handler? As I could see you could access the same objects in AM using `app_manager_` reference + public getters

Reply:
Fixed in 942e36845e9eca9cf6390731be80a8d155e7ecf9.